src/main/java/heronarts/lx/midi/surface/APC40Mk2.java

```diff
@@ -1067,6 +1067,9 @@ private void noteReceived(MidiNote note, boolean on) {
           // Make it the focusColor (i.e., turning the CUE LEVEL knob will tweak it), and
           // if there's a color on the clipboard, paste it here.
           LXSwatch swatch = getSwatch(MASTER_SWATCH);
+          if (index > swatch.colors.size()-1 && index < LXSwatch.MAX_COLORS) {
```

Comment:
I don't think this part belonged in this pull request?

Reply:
Hmm no it didn't.  I created this branch after the commit from my other pending pull request, perhaps that was the wrong move.  What's the best practice when staging multiple pull requests... branch each of them off the current dev branch?  I wasn't sure if doing that would require each of them to be rebased before merge and therefore more clunky on your end.  Still a little green on the git techniques.

Reply:
Yeah using separate branches off dev is generally the move and then pull-request from each, which contains one set of changes. GitHub will generally be clever about dealing with the rebasing stuff fairly automatically so long as there are no conflicts (which there wouldn't be in this case as they're touching separate files anyways).